w-books/src/redux/book/reducer.js

```diff
@@ -0,0 +1,58 @@
+const initialState = { loading: false, detail: null, comments: null, suggestions: null }
+
+const book = (state = initialState, action) => {
+  switch (action.type) {
+    case 'FETCH_BOOK':
```

Comment:
all these case statements look pretty similar. can you think of a way of avoiding repetition?